### PR TITLE
only append latest tag to image once

### PR DESCRIPTION
**Pull request by nkubala**
Also, use regex to check if image has tag. Previously, we were just doing a simple `strings.Contains(image, ":")`, which would give us a false positive when checking strings with the `remote://` or `daemon://` prefixes.  This should fix those cases.

### CODE REVIEW

File: pkg/util/image_utils.go [hunk 1]
```diff
@@ -67,3 +68,9 @@ func copyToFile(outfile string, r io.Reader) error {
 
 	return nil
 }
+
+// checks to see if an image string contains a tag.
+func HasTag(image string) bool {
+	tagRegex := regexp.MustCompile(".*:[^/]*$")
```
Comment (aaron-prindle) [inline pkg/util/image_utils.go]: can you add some unit tests for the regex?

File: cmd/root.go [hunk 2]
```diff
@@ -135,28 +135,21 @@ func getPrepperForImage(image string) (pkgutil.Prepper, error) {
 	}
 
 	// see if the image name has tag provided, if not add latest as tag
-	if !strings.Contains(image, ":") {
+	if !pkgutil.HasTag(image) {
 		image = image + ":latest"
```
Comment (tejal29) [inline cmd/root.go]: Create a Default for this

File: pkg/util/image_utils.go (hunk 1 above)
Comment (nkubala) [reply pkg/util/image_utils.go]: done

File: cmd/root.go (hunk 2 above)
Comment (nkubala) [reply cmd/root.go]: done

**Pull request merged**